Source/WebCore/page/PointerCaptureController.cpp

```diff
@@ -363,6 +364,11 @@ void PointerCaptureController::dispatchEvent(PointerEvent& event, EventTarget* t
     if (!target || event.target())
         return;
 
+    if (event.type() == eventNames().pointerdownEvent || event.type() == eventNames().pointerupEvent) {
+        if (auto* element = dynamicDowncast<HTMLElement>(target))
+            element->document().handlePopoverLightDismiss(event, *element);
+    }
```

Comment:
We should probably run this at the end of `pointerEventWillBeDispatched` per https://github.com/w3c/pointerevents/pull/460/files

Reply:
@rwlbuis To be clear, this should run last in `PointerCaptureController::pointerEventWillBeDispatched`.

Reply:
Sorry, I missed this comment earlier indeed, fixed.